ext/standard/string.c

```diff
@@ -1687,9 +1687,7 @@ PHP_FUNCTION(pathinfo)
    case insensitive strstr */
 PHPAPI char *php_stristr(char *s, char *t, size_t s_len, size_t t_len)
```

Comment:
An optional alternative: Maybe this function should be discouraged in a comment instead (use php_memnistr instead), or removed?

Reply:
> use php_memnistr instead if you aren't planning to mutate the string

Even if you're planning to mutate the string, the `zend_str_tolower` would be unnecessary with `php_memnstr`.